internal/pkg/aws/handlers/archer_user_agent.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package handlers
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/version"
+	"github.com/aws/aws-sdk-go/aws/request"
+)
+
+const UserAgentHeader = "Archer-User-Agent"
```

Comment:
I think this should just be "User-Agent"- that seems to be standard: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent

It's what I did in the fluent bit plugins: https://github.com/aws/amazon-kinesis-firehose-for-fluent-bit/blob/master/plugins/user_agent.go#L23

And what we did in ECS CLI v1: https://github.com/aws/amazon-ecs-cli/blob/master/ecs-cli/modules/clients/user_agent.go

Reply:
Couldn't there be a scenario where we create another user agent?

Reply:
Addressed